roles/satellite-clone/files/disassociate_capsules.rb

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/ruby
+username = "admin"
+password = "changeme"
```

Comment:
@sthirugn I'm not sure if I understand your concern - This script is meant to be used in the playbook only and its not exposed to the user in any way, so its a safe assumption that the credentials will be admin/changeme when the script is run. We also mention the credential change in the docs [here](https://github.com/RedHatSatellite/satellite-clone/blob/master/docs/satellite-clone.md#important-notes)